### ISSUE TITLE
After upgrading to 1.7.0 - TypeError: Cannot read property 'name' of undefined

### ISSUE THREAD

**Issue by bradbirnbaum** [benmosher/eslint-plugin-import#327]
```
13:12 $ eslint ./src
Cannot read property 'name' of undefined
TypeError: Cannot read property 'name' of undefined
    at EventEmitter.handleExportNamed (/usr/local/lib/node_modules/eslint-plugin-import/lib/rules/no-mutable-exports.js:80:56)
    at emitOne (events.js:95:20)
    at EventEmitter.emit (events.js:182:7)
    at NodeEventGenerator.enterNode (/usr/local/lib/node_modules/eslint/lib/util/node-event-generator.js:40:22)
    at CodePathAnalyzer.enterNode (/usr/local/lib/node_modules/eslint/lib/code-path-analysis/code-path-analyzer.js:607:23)
    at CommentEventGenerator.enterNode (/usr/local/lib/node_modules/eslint/lib/util/comment-event-generator.js:97:23)
    at Controller.traverser.traverse.enter (/usr/local/lib/node_modules/eslint/lib/eslint.js:886:36)
    at Controller.__execute (/usr/local/lib/node_modules/eslint/node_modules/estraverse/estraverse.js:397:31)
    at Controller.traverse (/usr/local/lib/node_modules/eslint/node_modules/estraverse/estraverse.js:501:28)
    at Controller.Traverser.controller.traverse (/usr/local/lib/node_modules/eslint/lib/util/traverser.js:36:33)
```

Note I'm using "eslint-config-airbnb": "9.0.1"

**Comment by vdh** [comment]
That looks like #317 I think?

**Comment by jfmengels** [comment]
Yes, this should be the same as #317. It's been fixed in https://github.com/benmosher/eslint-plugin-import/pull/322, but not released yet.

**Comment by bradbirnbaum** [comment]
Glad its been fixed.  Any ETA on a release?

**Comment by benmosher** [comment]
I'd guesstimate Thursday morning. Might be early next week or this weekend, though.

**Comment by benmosher** [comment]
Just kidding, just released it with 1.8.0. 😎

**Comment by bradbirnbaum** [comment]
Thanks
